crates/bevy_pbr/src/lib.rs

```diff
@@ -130,14 +130,12 @@ use bevy_ecs::prelude::*;
 use bevy_image::Image;
 use bevy_render::{
     alpha::AlphaMode,
-    camera::{CameraUpdateSystem, Projection},
+    camera::{sort_cameras, CameraUpdateSystem, Projection},
     extract_component::ExtractComponentPlugin,
     extract_resource::ExtractResourcePlugin,
-    render_asset::prepare_assets,
     render_graph::RenderGraph,
     render_resource::Shader,
     sync_component::SyncComponentPlugin,
-    texture::GpuImage,
```

Comment:
you should keep those two import

Reply:
Why? They're not used in the file anywhere...

Reply:
they were imported in the lib.rs file which was putting it in scope for other files